### ISSUE TITLE
Only "Help" button after starting up on windows 10.

### ISSUE THREAD

**Issue by B45i** [arduino/arduino-pro-ide#7]
**Only a help button in the title bar which contains toggle developer options**
After I extracted the zip file and execute the `Arduino-PoC.exe`, there is nothing on the screen,
Toggle developer option is available although it says developer tools is disconnected.

**Screenshots**
![image](https://user-images.githubusercontent.com/12771709/67148276-a2769f80-f2bb-11e9-8437-62548dd4f56f.png)

Also tried to start it from terminal this was the output:

```root INFO Theia app listening on http://localhost:50789.
daemon INFO >>> Starting 'arduino-cli' daemon... [C:\Users\B45i\Downloads\Arduino IDE Pro\resources\app\node_modules\arduino-ide-extension\build\arduino-cli.exe]
daemon INFO [INFO] 'C:\Users\B45i\Downloads\Arduino' is not recognized as an internal or external command,
daemon INFO [INFO] operable program or batch file.
daemon INFO [INFO] Daemon exited with code: 1. Signal was: null.
root INFO Error: Command failed: C:\Users\B45i\Downloads\Arduino IDE Pro\resources\app\node_modules\arduino-ide-extension\build\arduino-cli.exe daemon -v --log-level info --format json --log-format json
'C:\Users\B45i\Downloads\Arduino' is not recognized as an internal or external command,
operable program or batch file.

    at ChildProcess.exithandler (child_process.js:299:12)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:962:16)
    at Socket.stream.socket.on (internal/child_process.js:381:11)
    at Socket.emit (events.js:182:13)
    at Pipe._handle.close (net.js:606:12)
daemon INFO <<< The 'arduino-cli' daemon is up an running.
```

**Desktop (please complete the following information):**
 - OS: Winsows 10 pro
 - Version : Version	10.0.18362 Build 18362

**Comment by kittaakos** [comment]
@B45i, thank you for the detailed bug report. It helped a lot 👍

As a quick workaround; please remove the spaces from the `Arduino IDE Pro` folder name. The whitespaces in the path cause the issue. 😕

**Comment by kittaakos** [comment]
We have fixed the issue in `0.0.2`. I am closing it.

Please reopen if required.
